libdnf/CMakeLists.txt

```diff
@@ -169,13 +169,13 @@ if (GOBJECT_INTROSPECTION_FOUND)
                 DESTINATION share/gir-1.0
                 COMPONENT libraries)
         install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${GIR_TYPELIB}
-                DESTINATION ${LIB_INSTALL_DIR}/girepository-1.0
+                DESTINATION ${CMAKE_INSTALL_FULL_LIBDIR}/girepository-1.0
```

Comment:
Why are we using absolute paths here instead of relative ones? Shouldn't relative ones work fine?